src/main/java/fi/aalto/cs/apluscourses/intellij/notifications/CourseVersionTooNewError.java

```diff
@@ -0,0 +1,17 @@
+package fi.aalto.cs.apluscourses.intellij.notifications;
+
+import static fi.aalto.cs.apluscourses.utils.PluginResourceBundle.getText;
+
+import com.intellij.notification.Notification;
+import com.intellij.notification.NotificationType;
+import fi.aalto.cs.apluscourses.intellij.services.PluginSettings;
+
+public class CourseVersionTooNewError extends Notification {
```

Comment:
I was thinking of a better name and came up with "indated" as the opposite of outdated, but apparently that's not an english word :frowning_face:

Reply:
`CourseVersionBackwardsIncompatibleError` specific but too verbose for my liking